components/AboutTitleBar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="AboutTitleBar">
+    <div class="Background-Wrapper">
+      <div class="header-container">
+        <h2>
+          <span class="headertitle">
+            About the Bay Area Pandemic Dashboard
+          </span>
+        </h2>
+        <p>
+          <span class="headersubtitle">
+            Local data and information associated with the coronavirus pandemic
+          </span>
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+export default {
+  name: 'AboutTitleBar'
+}
+</script>
+<style lang="scss">
+.AboutTitleBar {
+  .Background-Wrapper {
+    background: url('../static/titlebar_bg.png') center no-repeat;
+    height: 340px;
+    position: relative;
+    width: 1440px;
+    height: 340px;
+    left: 0px;
+    right: 0px;
+    top: 0px;
```

Comment:
- You have height twice here.

- By default position relative would put Background-Wrapper in top 0, left 0. I believe you can remove left, right, top and it should still be exactly like how you have it.

- Your Background-wrapper is not responsive. When you resize the screen to be smaller than 650px, the text start to get cut off.
One way to make this responsive is:
width: 100%;
display: flex;
justify-content: center;
align-items: center;
remove .header-container position absolute, left, top

Reply:
background-wrapper was updated but I did not take out the header-container code. Whenever I did it, it adjusted both header and subheader towards the center